datasets/covid19_tracking/city_level_cases_and_deaths/city_level_cases_and_deaths_dag.py

```diff
@@ -66,16 +66,34 @@
         skip_leading_rows=1,
         write_disposition="WRITE_TRUNCATE",
         schema_fields=[
-            {"name": "date", "type": "DATE"},
-            {"name": "state", "type": "STRING"},
+            {"name": "date", "type": "DATE", "description": "Date of the observations"},
+            {
+                "name": "state",
+                "type": "STRING",
+                "description": "2-letter postal abbreviation for the state",
+            },
             {"name": "location", "type": "STRING"},
-            {"name": "city_or_county", "type": "STRING"},
-            {"name": "cases_total", "type": "INTEGER", "mode": "NULLABLE"},
+            {
+                "name": "city_or_county",
+                "type": "STRING",
+                "description": "Category of the location",
+            },
+            {
+                "name": "cases_total",
+                "type": "INTEGER",
+                "mode": "NULLABLE",
```

Comment:
Nit: NULLABLE is the default, so it can be omitted. Or do you intend for the columns where there is no "mode" to be "REQUIRED"?

Reply:
True, there are inconsistencies there (my bad!). I'll go with being explicit but make it consistent, i.e. include `mode` in every column for clarity.

Please see eed554f and https://github.com/GoogleCloudPlatform/public-datasets-pipelines/pull/7/commits/d1de1fdd47b2b3e0c885ca826f155d941c508fd6.